supporting_content/columns/chargefrequency.md

```diff
@@ -72,5 +72,5 @@ Current values observed in billing data for various scenarios:
   - Fees are important for cost allocation / amortization - sometimes needs to be isolated from other cost
   - How would recurring refunds or credits be handled? Like AWS MAP credits
 - AWS handling for SPs might be a concern - Anniversary charge (BillType) Savings Plan for $1 and a negation for UsageType (for -0.30)
-- What Charge Types can BE recurring? Would usage under free tier be recurring since every month that charge is $0 or would it be usage based?
+- What Charge Categorys can BE recurring? Would usage under free tier be recurring since every month that charge is $0 or would it be usage based?
```

Comment:
```suggestion
- What Charge Category values can BE recurring? Would usage under free tier be recurring since every month that charge is $0 or would it be usage based?
```